hl7/containers.py

```diff
@@ -73,20 +73,260 @@ def __getslice__(self, i, j):
         return self.__getitem__(slice(i, j))
 
     def __str__(self):
-        """Join a the child containers into a single string, separated
+        return self.separator.join((str(x) for x in self))
+
+
+class Builder(object):
```

Comment:
Suggest naming `class BuilderMixin:` to make more clear that it is a mixin

Reply:
A rose is a rose...

Renamed